Python/crossfire/crossfire/parser.py

```diff
@@ -82,39 +72,27 @@ def from_response(cls, response):
         return cls(**kwargs)
 
 
-def parse_response(method):
-    async def wrapper(self, *args, **kwargs):
-        """Converts API response to a dictionary, Pandas DataFrame or GeoDataFrame."""
-        format = kwargs.pop("format", None)
-        if format and format not in FORMATS:
-            raise UnknownFormatError(format)
+def parse_response(response, format=None):
```

Comment:
OK, vc limitou a ação do `parse_response` a apenas receber a resposta e reornar no formato solicitado, junto com `metadata`...
Acho que está alinhado com o meu primeiro comentário,

Reply:
Vi que foram feitas alterações nos testes, também. Imagino que adequando conforme as mudanças feitas. Mas aqui sinaliza que tem alguma coisa que não está passando na checagem...

Reply:
Algo relacionado ao [numpy](https://github.com/FelipeSBarros/crossfire/actions/runs/6658762081/job/18099346520?pr=69#:~:text=Note%3A%20This%20error%20originates%20from%20the%20build%20backend%2C%20and%20is%20likely%20not%20a%20problem%20with%20poetry%20but%20with%20numpy%20(1.25.2)%20not%20supporting%20PEP%20517%20builds.%20You%20can%20verify%20this%20by%20running%20%27pip%20wheel%20%2D%2Duse%2Dpep517%20%22numpy%20(%3D%3D1.25.2)%22%27.)

Reply:
> Algo relacionado ao numpy

Não exatamente. Detalhei em #70

Reply:
> Vi que foram feitas alterações nos testes, também

Sim. Pois antes o `parse_response` dependia do cliente (o `parse_response` que _fazia_ a requisição HTTP). Agora ele só trata a resposta (seja lá quem for que faz a requisição HTTP, obtém a resposta dessa requisição e aí sim passa essa resposta para o `parse_response`).